jax/experimental/jet.py

```diff
@@ -59,6 +59,8 @@ def jet_fun(primals, series):
   with core.new_master(JetTrace) as master:
     out_primals, out_terms = yield (master, primals, series), {}
     del master
+  out_terms = [tree_map(lambda x: onp.zeros_like(x, dtype=onp.result_type(out_primals[0])), series[0])
```

Comment:
We can also use ad_util.zeros_like_jaxval, but it'll do essentially the same thing. Easy to tweak later if we want to.

Reply:
Thanks for pointing this out! That'd be cleaner and more similar to what's done for `jvp`.

However, I needed to set `dtype` to be the same as `out_primals` and not `series` to pass tests for primitives with `bool` output. In these cases the `out_terms` needed to be instantiated as `False` instead of `0` (though oddly enough tests only fail on `float64`).